conda_env/env.py

```diff
@@ -51,7 +51,9 @@ def from_environment(name, prefix, no_builds=False):
     if len(pip_pkgs) > 0:
         dependencies.append({'pip': ['=='.join(a.rsplit('-', 2)[:2]) for a in pip_pkgs]})
 
-    channels = config.get_rc_urls()
+    # conda uses ruamel_yaml which returns a ruamel_yaml.comments.CommentedSeq
+    # this doesn't dump correctly using pyyaml
```

Comment:
> this doesn't dump correctly using pyyaml

I think this is the key issue.  There's no problem with this PR.  But I think we also need to force `ruamel_yaml` or `ruamel.yaml` instead of `pyyaml` for conda-env now that we're using it with conda.